### PR TITLE
IDP: Move IDP tutorials to its own section under Tutorials

### PR DESCRIPTION
**Pull request by OrkoHunter**
Preview URL - https://64abf4abac29f80e82bf6ac6--harness-developer.netlify.app/

### CODE REVIEW

**Comment by ravilach** [comment]
@wei-harness not sure if we have an updated tutorial banner for IDP. @OrkoHunter got really far!

**Comment by OrkoHunter** [comment]
@wei-harness Would love to talk about some graphics that can be built for IDP. cc @Tranimal-N

**Comment by OrkoHunter** [comment]
IMO let's do the design work async and not block this PR.

File: tutorials/internal-developer-portal/create-your-first-service-onboarding-pipeline.md [hunk 1]
```diff
@@ -15,20 +15,25 @@ Your users (developers) must perform a sequence of tasks to create the applicati
...
+
 To create a Build or Custom stage, perform the following steps:
-1. In the sidebar of the Harness application, select **Projects**, and then select a project. 
-  
-  You can also create a new project for the service onboarding pipelines. Eventually, all the users in your account should have permissions to execute the pipelines in this project. For information about creating a project, go to [Create organizations and projects](https://developer.harness.io/docs/platform/organizations-and-projects/create-an-organization/).
 
-2. Select **Pipelines**, and then select **Create a Pipeline**. 
+1. In the sidebar of the Harness application, select **Projects**, and then select a project.
+
+You can also create a new project for the service onboarding pipelines. Eventually, all the users in your account should have permissions to execute the pipelines in this project. For information about creating a project, go to [Create organizations and projects](https://developer.harness.io/docs/platform/organizations-and-projects/create-an-organization/).
```
Comment (deepakbolangady) [inline tutorials/internal-developer-portal/create-your-first-service-onboarding-pipeline.md]:
```suggestion
You can also create a new project for the service onboarding pipelines. Eventually, all the users in your account should have permissions to execute the pipelines in this project. For information about creating a project, go to [Create organizations and projects](/docs/platform/organizations-and-projects/create-an-organization/).
```
Reason for suggested change: an absolute URL causes a new tab to open. IMO, this behavior is OK, and, indeed, desirable for URLs to other websites. For internal (HDH) pages, we should use the part of the URL that follows "https://developer.harness.io" in the URL of the target page on the public website (or http://localhost:3000, if using a local instance). That way, the target page opens in the same tab. There is at least one other absolute URL that points to an HDH article on this page. It's target is https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/. Good to replace with recommended stub for consistency across HDH.

**Review by deepakbolangady — APPROVED** [review]
Replaced absolute URLs to internal (HDH) articles with stubs so that their behavior is consistent with other internal references. Approving.

**Comment by OrkoHunter** [comment]
Yes indeed, thanks Deepak! Relative urls are best ✅

**Pull request merged**